test/input/sf-light/sf-light-1k.conf

```diff
@@ -9,7 +9,7 @@ include "../common/matsim.conf"
 # Agentsim
 ##################################################################
 beam.agentsim.simulationName = "sf-light-1k-xml"
-beam.agentsim.agentSampleSizeAsFractionOfPopulation = 0.001
+beam.agentsim.agentSampleSizeAsFractionOfPopulation = 1.0
```

Comment:
Revert the change from here https://github.com/LBNL-UCB-STI/beam/pull/3215/files#diff-fcd2a00c693c8e1382cda931e2204f3535b628669b88ad3338afd919c935bf91R12